api/breaker/breaker.go

```diff
@@ -103,7 +103,7 @@ func (s State) String() string {
 
 // ErrStateTripped will be returned from executions performed while the CircuitBreaker
 // is in StateTripped
-var ErrStateTripped = trace.ConnectionProblem(nil, "breaker is tripped")
+var ErrStateTripped = trace.ConnectionProblem(nil, "circuit breaker is tripped. unable to connect to api service")
```

Comment:
I don't know if this is an improvement. If I saw this error I would be asking myself what "api service" is.

Reply:
Let me see if we can give better context.